pkg/generate/cspc.go

```diff
@@ -193,32 +198,36 @@ func makePools(poolType string, nDevices int, bd map[string][]v1alpha1.BlockDevi
 				return nil, fmt.Errorf("not enough eligible blockdevices found on node %s, want %d, found %d", nodes[i], nDevices, len(bds))
 			}
 			index := 0
-			for d := 0; d < nDevices/2; d++ {
-				raids := []cstorv1.CStorPoolInstanceBlockDevice{{BlockDeviceName: bds[index].Name},
-					{BlockDeviceName: bds[index+1].Name}}
```

Comment:
Q: Do we need to log some info when `nDevices%min != 0`? Informing only `x` devices has picked for pool creation on so and so host?

Reply:
Right, this makes sense, I can just add this check assuming minimum number of disks per RAID-group. As, adding more and more BDs than the minimum will just increase the replication and not the pool-size.